projects/metallb/project.yaml

```diff
@@ -0,0 +1,11 @@
+homepage: "https://metallb.universe.tf"
+language: go
+primary_contact: "fpaoline@redhat.com"
+auto_ccs:
+  - rbryant@redhat.com
```

Comment:
Please replace russell with obraunsh@redhat.com, as he's more active on the project.
Thanks!